sonic3k.asm

```diff
@@ -25374,7 +25374,7 @@ loc_131F2:
 		move.w	$A(a0),(Saved_art_tile).w
 		move.w	$46(a0),(Saved_solid_bits).w
 		move.l	#Obj_Tails2P_Tail,(Super_stars).w
-		move.w	a0,($FFFFCBF0).w
+		move.w	a0,(Tails_tails_2P+$30).w
```

Comment:
Suggestion for future enhancement: maybe add a new 'parent' constant, equal to $30, and change these to (Tails_tails_2P+parent).w (and likewise for the others).

Reply:
https://github.com/fredbronze/skdisasm/blob/0316b5db61548dece88d501f3b68193e409fc07e/sonic3k.asm#L28202-L28204

S3K already has a 'parent' constant ($42), this one is a holdover from S2. Quite simply it is a mess; there are at least five different offsets which serve the same purpose between all the objects, and I'm not sure they're worth documenting -- after all, if you want to know how they're used, you'll need to read the child object's code anyway.